actionview/test/template/tag_helper_test.rb

```diff
@@ -37,6 +37,24 @@ def test_tag_options
     assert_match(/class="elsewhere"/, str)
   end
 
+  def test_tag_options_with_array_of_numeric
+    str = tag(:input, value: [123, 456])
```

Comment:
Does `class: [123, 456]` and `class: [klass.new]` still work?

Reply:
Ah indeed, I see what mean. Nop it doesn't work, let me fix that

Reply:
Now it should be good